src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,36 @@
+package commands;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import tasks.Task;
+
+public class DeleteCommand extends Command {
+    int taskNumToDelete;
+
+    public DeleteCommand(int taskNumToDelete) {
+        this.taskNumToDelete = taskNumToDelete;
+    }
+
+    /**
+     * This method is used to delete one item from the task list
+     * after user request.
+     *
+     * @return duke response after deletion
+     */
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
+        assert Math.abs(taskNumToDelete) >= 0;
+        try {
+            Task t = tasks.deleteTask(taskNumToDelete);
+            assert tasks.getList()!= null;
+            storage.updateList(tasks.getList());
+            return ("Noted. I've removed this task: \n" + t + "\nNow you have "
```

Comment:
It may be better to put the return statement outside the "try .. catch" block to follow the coding standard.